gap_follow:
  ros__parameters:
    debug: 1
    kp: 0.6
    kd: 0.7
    ki: 0.0
    max_dist_threshold: 10.0
    max_speed: 20.0
    car_radius: 0.3
    angle_range: [1.0,  10.0, 20.0, 30.0, 40.0, 60.0, 80.0]
    speed_range: [8.0,   6.0,  2.0,  1.0,  1.0,  1.0,  1.0]
    drivable_angle: 110.0
